modules/customize-certificates-manually-rotate-service-ca.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-signing-certificates.adoc
+
+[id="manually-rotate-service-ca-{context}"]
+= Manually rotate the service CA certificate
+
+The service CA is valid for one year after {product-title} is 
+installed. If you have a cluster that has been running
+for one year without upgrades, then you must manually
+refresh the service CA.
+
+This process is automatically performed during an upgrade.
+Follow these steps only if you do not update your cluster.
+
+.Prerequisites
+
+* You must be logged in as a cluster admin.
+
+.Procedure
+
+. View the expiration date of the current service CA certificate by
+using the following command. 
++
+----
+$ oc get secrets/signing-key -n openshift-service-ca \
+     -o template='{{index .data "tls.crt"}}' \
+     | base64 -d \
+     | openssl x509 -noout -enddate
+----
++
+If this certificate expires before a scheduled upgrade, then proceed 
+through the following steps. Otherwise, there is no need to perform 
+the rotation at this time.
+
+. Manually rotate the service CA. This process generates a new service CA
+which will be used to sign the new service certificates.
++
+----
+$ oc delete secret/signing-key -n openshift-service-ca
+----
+
+. To apply the new certificates to all services, restart all the Pods 
+in your cluster. This command ensures that all services use the 
+updated certificates.
++
+----
+$ for I in $(oc get ns -o jsonpath='{range .items[*]} {.metadata.name}{"\n"} {end}'); \ 
```

Comment:
Won't this by any chance irreversibly kill all the pods that are not associated to any replica controller/set?

Reply:
If it's not associated with a replica set it would otherwise still be backed by a deployment, though

Reply:
Resolving this conversation for now.

Reply:
What I meant by "replicasets" were even those created by deployments internally, meaning - you can always run a pod by itself. It may not be all that useful, but there can always be some pods that run just like that, can't they?